src/options.ts

```diff
@@ -11,6 +12,8 @@ export interface TerminalScreenshotOptions {
   readonly backgroundColor: string;
   // Type of the screenshot to be generated. (default: png)
   readonly type: "png" | "jpeg";
+  // Name of the theme to use. (default: "")
+  readonly theme: ITheme; // values https://github.com/ysk2014/xterm-theme/blob/master/src/index.js
 }
 
```

Comment:
I suggest not mentioning or depending on `xterm-theme` (here + `README` + others), as it is unmaintained, and without proper docs/policy in place. The users can always refer to the official `xterm` package if needed.

Reply:
Perhaps we can refer to the example this PR will be adding to our `main` branch instaed of the unknown/unmaintained repo? For example:

- https://github.com/OmarTawfik/terminal-screenshot/blob/main/tests/ayu.json